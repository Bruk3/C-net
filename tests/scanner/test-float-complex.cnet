1.
0.5e-15
.3e+3
.2
1e5
3.5e-4