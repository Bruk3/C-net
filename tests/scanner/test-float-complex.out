FLOATLIT(1.000000)
FLOATLIT(0.000000)
FLOATLIT(300.000000)
FLOATLIT(0.200000)
FLOATLIT(100000.000000)
FLOATLIT(0.000350)
EOF